preflight.sh

```diff
@@ -0,0 +1,36 @@
+echo "Running preflight.sh"
+# Command Flags:
+# PLATFORM (Required, must be "gke" or "gce")
+#
+# Example to invoke this script:
+# bash preflight.sh PLATFORM=gke
+
+# Stop execution if any command exits with error
+set -e
+
+# Set environment variables
+for ARGUMENT in "$@"; do
+    IFS='=' read -r KEY VALUE <<< "$ARGUMENT"
+    export "$KEY"="$VALUE"
+done
+
+if [ -z $PLATFORM ]; then
+    echo "Error: PLATFORM flag is missing."
+    exit 1
+fi
+
+if [[ $PLATFORM == "gce" ]]; then
+    # Set up network for running on gce
+    echo "Setting up network for GCE"
+    bash rto_setup.sh
+    # For DNS lookup when running on large number of VMs on gce
+    echo '142.250.123.95 www.googleapis.com' | sudo tee -a /etc/hosts
```

Comment:
@rwitten is this still needed? I vaguely recall this being fixed at some point but maybe not

Reply:
Ya I think we should get rid of this IP pinning for now. It is needed only in a very weird setting.

Reply:
Sounds good, I removed it.